GSA-TTS/FAC
# Upload Page Instructions/Template Download

# Several Changes from Frontend Review pt. 2

Issue(s):
Initial ticket (closed early) - https://github.com/GSA-TTS/FAC/issues/1899
Therefore it falls under - https://github.com/GSA-TTS/FAC/issues/1560

Figma: [Link](https://www.figma.com/file/lnkXJTfKzD3YBhiJZ2QFEv/FAC-Source-of-Truth?node-id=1%3A31587&mode=dev)

## Changes:
1. Upload page
    * Links to the instructions and the template download have been added
    * "Optional" workbooks, the ones that may have no content if the user has no findings, get special extra text to let them know they still need to put in a blank workbook
2. Cross validation
    * Improve the ready for certification button, so they can't get ahold of the ready for cert links early
    * Improved back buttons
4. Audit access - remove references to emailing anybody
5. Checklist - copy updates to remove references to not having to upload a workbook

## How to test:
Switch to this branch and run normally. Navigate to or start a new submission. Verify that the listed changes have been made, particularly on the XLSX upload pages. They have links. 

## Screenshot:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/c393a2e3-a409-4610-9b8e-f22fb45ab9c1)
![image](https://github.com/GSA-TTS/FAC/assets/91098850/876f1cad-d3c3-4329-8ad4-e1a88c60e349)


## PR checklist: submitters


## PR checklist: reviewers

        
The larger the PR, the stricter we should be about these points.
